modules/update-duration-mco.adoc

```diff
@@ -50,4 +50,4 @@ ip-10-0-207-224.us-east-2.compute.internal  Ready                       worker
 +
 If the status of the node is `NotReady` or `SchedulingDisabled`, then the node is not available and this impacts the update duration.
 +
-You can check the status of nodes from the *Administrator* perspective in the web console by expanding **Compute** → **Node**.
+You can check the status of nodes from the *Administrator* perspective in the web console by expanding **Compute** → **Nodes**.
```

Comment:
```suggestion
You can check the status of nodes from the *Administrator* perspective in the web console by expanding **Compute** **>** **Nodes**.
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=elements-menus-navigation#navigation-trees
I remember a discussion about using the arrow, but I can't find anything to support it. Let me know if you find guidance for this. Also, I am not sure if this will work; it should:
**Compute > Nodes**